### DISCUSSION TITLE
Welcome to Virtual Network Integration Recipes Discussions!

### DISCUSSION THREAD

**Discussion by mcollier** [Azure-Samples/virtual-network-integration-recipes#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hannesne** [comment]
Hi there! Looking forward to collaborating some more.

**Comment by moniquema** [comment]
Hi, when I tried to create cert based on create-certificates.sh, it's not work.
Error as :
$ ./create-certificates.sh *.toolkitDev.net
Creating root CA certificate.
name is expected to be in the format /type0=value0/type1=value1/type2=... where characters may be escaped by \. This name is not in that format: 'C:/Program Files/Git/CN=Sample CA'
problems making Certificate Request
Can't open rootCA.csr for reading, No such file or directory
14760:error:02001002:system library:fopen:No such file or directory:../openssl-1.1.1q/crypto/bio/bss_file.c:69:fopen('rootCA.csr','r')
14760:error:2006D080:BIO routines:BIO_new_file:no such file:../openssl-1.1.1q/crypto/bio/bss_file.c:76:
base64: rootCA.crt: No such file or directory
Creating server certificate for *.toolkitDev.net.
Can't open /proc/1848/fd/63 for reading, No such file or directory
8096:error:02001003:system library:fopen:No such process:../openssl-1.1.1q/crypto/bio/bss_file.c:69:fopen('/proc/1848/fd/63','r')
8096:error:2006D080:BIO routines:BIO_new_file:no such file:../openssl-1.1.1q/crypto/bio/bss_file.c:76:
Can't open domain.crt for reading, No such file or directory
4500:error:02001002:system library:fopen:No such file or directory:../openssl-1.1.1q/crypto/bio/bss_file.c:69:fopen('domain.crt','r')
4500:error:2006D080:BIO routines:BIO_new_file:no such file:../openssl-1.1.1q/crypto/bio/bss_file.c:76:
base64: domain.pfx: No such file or directory
